src/includes/getting-started-primer/apple.mdx

```diff
@@ -18,31 +18,12 @@
   - System events (battery level or state changed, memory warnings, device orientation changed, keyboard did show and did hide, screenshot taken)
   - Outgoing HTTP requests
 - [Release Health](/product/releases/health/) tracks crash free users and sessions
+- Automatic <PlatformLink to="/performance/instrumentation/">Performance Tracking</PlatformLink>
+  - Rendering of UIViewControllers
+  - Performance of HTTP requests
+  - Distributed tracing
+  - [Mobile Vitals](https://blog.sentry.io/2021/08/23/mobile-vitals-four-metrics-every-mobile-developer-should-care-about)
```

Comment:
Only suggestion would be linking to our docs on mobile vitals vs a blog post.
https://docs.sentry.io/product/performance/mobile-vitals/

Reply:
```suggestion
  - [Mobile Vitals](https://docs.sentry.io/product/performance/mobile-vitals/)
```